frontend/src/hooks/infraMonitoring/useGetK8sClustersList.ts

```diff
@@ -38,14 +40,15 @@ export const useGetK8sClustersList: UseGetK8sClustersList = (
 			return options.queryKey;
 		}
 
-		return [REACT_QUERY_KEY.GET_CLUSTER_LIST, requestData];
-	}, [options?.queryKey, requestData]);
+		return [REACT_QUERY_KEY.GET_CLUSTER_LIST, requestData, dotMetricsEnabled];
+	}, [options?.queryKey, requestData, dotMetricsEnabled]);
```

Comment:
Does `dotMetricsEnabled` need to be a part of the query key? We add a variable to the query key if we want to refetch the data when its value changes. Is that required in our case?